receiver/prometheusreceiver/metrics_receiver_helper_test.go

```diff
@@ -82,11 +89,18 @@ func (mp *mockPrometheus) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 		rw.WriteHeader(404)
 		return
 	}
-	if pages[index].useOpenMetrics {
+	switch {
+	case pages[index].useProtoBuf:
+		rw.Header().Set("Content-Type", "application/vnd.google.protobuf; proto=io.prometheus.client.MetricFamily; encoding=delimited")
```

Comment:
@krajorama could you explain why is it `encoding=delimited`?

Reply:
TBH I started to type the first part and copilot suggested the proto and encoding parts :) I've asked @beorn7 to take a look if it's needed. IIRC tests pass fine without it.